### PR TITLE
Adjust Accordion styles

### PR DESCRIPTION
**Pull request by HenrikHovhannisyan**

### CODE REVIEW

File: src/pages/Recipes/index.tsx [hunk 1]
```diff
@@ -34,7 +34,14 @@ const Recipes = () => {
...
     const classes = useStyles();
-    const { addRecipeBtn } = classes;
+    const { 
+        addRecipeBtn,
+        btnRecipes,
+        addRecipe,
+        ingredients,
+        ingredientsContent,
+        recipeList,
+    } = classes;
```
Comment (SaulReznik) [inline src/pages/Recipes/index.tsx:44]: if the classes are to much, don't destruct them, just use directly from the classes object like this `classes.addRecipe`

File: src/pages/Recipes/index.tsx [hunk 2]
```diff
@@ -141,17 +152,19 @@ const Recipes = () => {
...
                         </ol>
                         <div>
-                            <span>Instructions:</span>
-                            <textarea
-                                value={newRecipe.instructions}
-                                onChange={onNewRecipeInstructionsChange}
-                            />
+                            <label>Instructions:
+                                <textarea
+                                    rows={5}
```
Comment (SaulReznik) [inline src/pages/Recipes/index.tsx:157]: Move this number to constants and use it from there like this:

**constants.ts**
`const DEFAULT_TEXTAREA_ROWS = 5`

**Recipes/index.tsx**
`rows={DEFAULT_TEXTAREA_ROWS}`

File: src/pages/Recipes/styles.ts [hunk 3]
```diff
@@ -18,7 +18,89 @@ const useStyles = createUseStyles({
...
+    },
+
+    recipeList: {
+      textAlign: 'start',
+      '& li': {
+        paddingLeft: 5,
+        '& ol': {
+          paddingLeft: 0
+        }
+      }
```
Comment (SaulReznik) [inline src/pages/Recipes/styles.ts:101]: don't use to much nested styles, instead use separated classnames

**Review by SaulReznik — CHANGES_REQUESTED** [review]

File: src/pages/Recipes/index.tsx (hunk 1 above)
Comment (HenrikHovhannisyan) [reply src/pages/Recipes/index.tsx:44]: Done

File: src/pages/Recipes/index.tsx (hunk 2 above)
Comment (HenrikHovhannisyan) [reply src/pages/Recipes/index.tsx:157]: Done

File: src/pages/Recipes/styles.ts (hunk 3 above)
Comment (HenrikHovhannisyan) [reply src/pages/Recipes/styles.ts:101]: Done

**Review by SaulReznik — APPROVED** [review]

**Pull request merged**